python-sdk/tests/sql/operators/test_dataframe.py

```diff
@@ -334,3 +334,43 @@ def sample_df_1(**kwargs):
 
     task = sample_df_1(output_table=output_table)
     assert task.operator.outlets == []
+
+
+def test_dataframe_list(sample_dag):
+    df1 = pandas.DataFrame({"numbers": [1, 2, 3], "colors": ["red", "white", "blue"]})
+    df2 = pandas.DataFrame({"numbers": [4, 5, 6], "colors": ["red", "white", "blue"]})
+
+    @aql.dataframe
+    def create_multiple_dataframes():
+        return [df1, df2]
+
+    @aql.dataframe()
+    def validate_dfs(input_dfs):
+        assert df1.equals(input_dfs[0])
+        assert df2.equals(input_dfs[1])
```

Comment:
Should we add/modify one of the example DAGs or docs - https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/astro/sql/operators/dataframe.rst to tell that we support multiple dataframes

Reply:
@kaxil added